src/components/Waveform/Waveform.tsx

```diff
@@ -432,10 +435,16 @@ export const Waveform = forwardRef<IWaveformRef, IWaveform>((props, ref) => {
         const clampedSeekAmount = clamp(seekAmount, 0, 1);
 
         if (!panMoving) {
-          seekToPlayer({
-            playerKey: `PlayerFor${path}`,
-            progress: clampedSeekAmount * songDuration,
-          });
+          try {
+            await seekToPlayer({
+              playerKey: `PlayerFor${path}`,
+              progress: clampedSeekAmount * songDuration,
+            });
+          } catch (e) {
+            // If the player is not prepared, triggering the stop will reset the player for next click
+            await stopPlayerAction();
```

Comment:
Hi @dprevost-LMI What is the purpose of calling stopPlayerAction() in this context, especially if seekToPlayer() fails? I ask because it resets the current progress value to 0, which could lead to incorrect values being reported in the onCurrentProgressChange callback